docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,176 @@
+# Commissioning nRF Connect Accessory using Android CHIPTool
+
+This article describes how to use [CHIPTool](../../src/android/CHIPTool/README.md) for Android smartphones to commission a Nordic Semiconductor nRF 52840 DK with installed
+[nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md) onto a CHIP-enabled Thread network.
+
+Prerequisites:
+- 2 x nRF 52840 DK
+- 1 x smartphone with Android 8+
+- 1 x PC with Ubuntu 20.04 and Docker installed (however, all the presented steps should be achievable in some way using other popular operating systems)
+
+The essence of the commissioning process is composed of the following steps:
+1. CHIPTool discovers the accessory device over BLE.
+2. CHIPTool establishes a secure connection to the device over BLE and sends Thread credentials.
```

Comment:
as well as CHIP operational credentials right? (at least in current implementation some key is exchanged for later use in AES-CCM)

Reply:
yes, I can mention that. I just didn't want to go into details in the tutorial.